src/main/java/org/sefglobal/scholarx/service/SimpleJavaMail.java

```diff
@@ -0,0 +1,30 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.email.EmailBuilder;
+import org.simplejavamail.mailer.MailerBuilder;
+
+public class SimpleJavaMail {
+
+    public void sendEmail(String fname,String emailAddress, String subject, String message, boolean showButton) {
+
+        String username = "pasindur2@gmail.com";
+        String password = "wwooravkrkhorobx";
```

Comment:
let's remove this

Reply:
I should find a way to use those as environment variables. can you suggest a way of doing that? @anjula-sack